Dockerfile

```diff
@@ -5,13 +5,18 @@ MAINTAINER KBase Developer
 # Insert apt-get instructions here to install
 # any required dependencies for your module.
 
-# RUN apt-get update
+RUN apt-get update && \
+    apt-get install bioperl && \
+    apt-get install uuid-runtime
+
 RUN cpanm -i Config::IniFiles && \
     cpanm -i UUID::Random && \
     cpanm -i HTML::SimpleLinkExtor && \
     cpanm -i WWW::Mechanize --force && \
     cpanm -i MIME::Base64 && \
-    apt-get -y install nano
+    cpanm -i Text::Trim && \
+    cpanm -i URI::Encode && \
+    cpanm -i Test::Most
```

Comment:
It would be better to use a `cpanfile` for installing Perl dependencies. I can show you how to do this if you like.